docs/installing.rst

```diff
@@ -47,7 +53,9 @@ To build hipRAND for the CUDA platform instead, the following software is requir
 Obtaining Sources
 ^^^^^^^^^^^^^^^^^
 
-The hipRAND sources are available from the `hipRAND GitHub Repository <https://github.com/ROCmSoftwarePlatform/hipRAND>`_. Use the branch that matches the system-installed version of ROCm. For example on a system that has ROCm 5.4 installed, use the following command to obtain hipRAND sources::
+The hipRAND sources are available from the `hipRAND GitHub Repository <https://github.com/ROCmSoftwarePlatform/hipRAND>`_. Use the branch that matches the system-installed version of ROCm. For example on a system that has ROCm 5.4 installed, use the following command to obtain hipRAND sources:
```

Comment:
```suggestion
The hipRAND sources are available from the `hipRAND GitHub Repository <https://github.com/ROCm/hipRAND>`_. Use the branch that matches the system-installed version of ROCm. For example on a system that has ROCm 5.4 installed, use the following command to obtain hipRAND sources:
```

Since we're updating the text, we should use the consolidated ROCm org

Reply:
Done as requested (multiple occurrences).